<launch>
	<node pkg="joy" type="joy_node" name="joy_node" output="screen" clear_params="true" required="true"/>
	<node pkg="joy_mapper_sean" type="joy_mapper_node.py" name="joy_mapper" output="screen" clear_params="true" required="true"/>
</launch>
